lib/ansible/modules/cloud/amazon/elasticache.py

```diff
@@ -393,17 +395,17 @@ def _requires_modification(self):
 
         # Check cache security groups
         cache_security_groups = []
-        for sg in self.data['CacheSecurityGroups']:
-            cache_security_groups.append(sg['CacheSecurityGroupName'])
+        for security_group in self.data['CacheSecurityGroups']:
```

Comment:
Not necessary to modify this, right? It can still be:
```
for sg in self.data['CacheSecurityGroups']:
    cache_security_groups.append(sg['CacheSecurityGroupName'])
```